src/Storefront/Resources/app/storefront/src/main.js

```diff
@@ -116,7 +115,6 @@ PluginManager.register('GallerySlider', GallerySliderPlugin, '[data-gallery-slid
 PluginManager.register('ProductSlider', ProductSliderPlugin, '[data-product-slider]');
 PluginManager.register('ZoomModal', ZoomModalPlugin, '[data-zoom-modal]');
 PluginManager.register('Magnifier', MagnifierPlugin, '[data-magnifier]');
-PluginManager.register('ImageZoom', ImageZoomPlugin, '[data-image-zoom]');
```

Comment:
Is this backward compatible?

Reply:
It should be i my opinion, because the ImageZoomPlugin works very closely together with the ZoomModalPlugin and cant't be really used without it.
And i didn't delete (or change) the ImageZoomPlugin itself. 
Only the registration and initialization is done now in the ZoomModalPlugin.